scripts/benchmarks/bench_quadruped_replicate_global_terrain.py

```
@@ -0,0 +1,398 @@
+            row["total_startup_median_seconds"]
+        )
+
+        for key in phase_keys:
+            base = key.replace("_median_seconds", "")
+            if mode == "legacy":
+                contribution_percent = 0.0
+            else:
+                if total_delta == 0.0:
+                    raise ValueError(
+                        f"total_startup delta is zero for non-legacy row at num_worlds={num_worlds}, mode={mode}"
+                    )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)